app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -62,29 +58,54 @@ class BrowserWebViewClient(
     @Suppress("OverridingDeprecatedMember")
     override fun shouldOverrideUrlLoading(view: WebView, urlString: String): Boolean {
         val url = Uri.parse(urlString)
-        return shouldOverride(view, url, RequestOrigin.Unknown)
+        return shouldOverride(view, url)
     }
 
     /**
      * API-agnostic implementation of deciding whether to override url or not
      */
-    private fun shouldOverride(webView: WebView, url: Uri, requestOrigin: RequestOrigin): Boolean {
-        return try {
-            Timber.d("Request origin: ${requestOrigin.javaClass.simpleName}, shouldOverride $url")
+    private fun shouldOverride(webView: WebView, url: Uri): Boolean {
+        try {
+            Timber.v("shouldOverride $url")
 
             val urlType = specialUrlDetector.determineType(url)
```

Comment:
Lint suggestion: `urlType` declaration can be moved into when

Reply:
Done!